gitlab/v4/cli.py

```diff
@@ -106,11 +106,12 @@ def do_custom(self) -> Any:
                 if TYPE_CHECKING:
                     assert isinstance(self.cls._id_attr, str)
                 data[self.cls._id_attr] = self.args.pop(self.cls._id_attr)
-            obj = self.cls(self.mgr, data)
-            method_name = self.action.replace("-", "_")
-            return getattr(obj, method_name)(**self.args)
+            class_instance = self.cls(self.mgr, data)
```

Comment:
I gotta be honest I spent 15 minutes thinking about what to name this thing. But I've had enough of `_obj` and `_cls` in here :sweat_smile: